src/pages/account/AdminPage.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import styled from "styled-components";
+
+import axiosApi from "../../utils/AxiosApi";
+import Card from "../../components/Card";
+import NavBar from "../../components/NavBar";
+import Button from "../../components/Button";
+
+const AdminPage = () => {
+  const [stories, setStories] = useState([]);
+  const [child, setChild] = useState({});
+  const [id, setId] = useState("");
+  const { childId } = useParams();
+
+  useEffect(() => {
+    const childData = async () => {
+      const response = await axiosApi(`/getChild/${childId}`);
+      const childData = await response.data;
+      const storyData = await response.data.stories;
+      setChild(childData);
+      setStories(storyData);
+    };
+    return childData();
```

Comment:
Why do you need here a `return`?

Reply:
without it I sometimes get an error. I don't know why yet

Reply:
What kind of error? Maybe you can get information in the error message why you get an error

Reply:
The useEffect error we were looking at the other day.